src/docker/DockerApi.ts

```diff
@@ -268,6 +269,7 @@ class DockerApi {
                 const optionsForBuild: Dockerode.ImageBuildOptions = {
                     t: imageName,
                     buildargs: buildargs,
+                    ...overrideOptions,
```

Comment:
Is this how it's passed to Dockerode? Isn't it using a key<>value pair structure?

Reply:
Great way to word that question; it isn't! I'll take a look at Dockerode and make sure I'm passing the variable correctly. :) Thank you for taking the time to review this for me!